src/phoenix/server/api/routers/v1/experiment_runs.py

```diff
@@ -37,7 +37,7 @@ async def create_experiment_run(request: Request) -> Response:
     output = payload["output"]
     start_time = payload["start_time"]
     end_time = payload["end_time"]
-    error = payload["error"]
```

Comment:
Output is a nullable type.

Reply:
Why does a nullable type affect our payload signature here? We can pass in null/none

Reply:
I just figured it's more natural to default to `None` for nullable types.